update-go-version.sh

```diff
@@ -12,14 +12,17 @@ echo "Updating golang version to $golang_base_version"
 go mod edit -go ${golang_base_version}
 go mod edit -go ${golang_base_version} tools/go.mod
 sed -i "s,^GOVERSION = 1.[0-9]\+,GOVERSION = ${golang_base_version}," Makefile
```

Comment:
If you change `GOVERSION` to 1.21.0 in the next commit, you need to change this line.